Release/cmake/cpprest_find_botan.cmake

```diff
@@ -0,0 +1,13 @@
+function(cpprest_find_botan)
+  if(TARGET cpprestsdk_botan_internal)
+    return()
+  endif()
+
+  # TODO: this does not make sure Botan is set and does not fail at configure time
```

Comment:
Either this todo is an actual todo in which case it should be fixed before we merge this, or it's not an actual todo in which case it should not be committted.